services/slam/builtin/builtin.go

```diff
@@ -636,13 +636,15 @@ func (slamSvc *builtIn) StartDataProcess(
 						}
 						if c != nil {
 							c <- 1
+							c = nil
```

Comment:
Setting the channel to nil after we write to it, so that we don't attempt to write to it again. Otherwise, we will hang trying to write to the channel, since the tests only read from it once.